spec/features/dashboard_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+feature 'dashboard' do
+  context 'authorized user' do
+    scenario 'can see other users successfully' do
+      user1 = FactoryGirl.create(:user)
+      user2 = FactoryGirl.create(:user)
+      sign_in_as(user2)
+      visit root_path
+
+      expect(page).to have_content(user1.email)
+      expect(page).to have_content(user2.email)
+    end
+
+    scenario 'can filter for looking to pair' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.